api/server/structs/endpoints_beacon.go

```diff
@@ -184,3 +184,15 @@ type WeakSubjectivityData struct {
 	WsCheckpoint *Checkpoint `json:"ws_checkpoint"`
 	StateRoot    string      `json:"state_root"`
 }
+
+type GetDepositSnapshotResponse struct {
+	Data *DepositSnapshot `json:"data"`
+}
+
+type DepositSnapshot struct {
+	Finalized            [][]string `json:"finalized"`
```

Comment:
This doesn't seem like the correct data type. https://github.com/ethereum/beacon-APIs/blob/d2f846e0e0d551973253df4b5341655eddf92f1d/types/api.yaml#L3

```suggestion
	Finalized            []string   `json:"finalized"`
```

Probably needs gofmt...